thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -551,6 +559,8 @@ service IConfigNodeRPCService {
    */
   TDataNodeRemoveResp removeDataNode(TDataNodeRemoveReq req)
 
+  TDataNodeUpdateResp updateDataNode(TDataNodeUpdateReq req)
```

Comment:
The response of updateDataNode should also be TDataNodeRegisterResp. Because there are configuration parameters in TDataNodeRegisterResp.